portal/templates/redesign/base_new.html

```diff
@@ -0,0 +1,215 @@
+{% load staticfiles %}
+{% load app_tags %}
+{% load cache %}
+
+{# CMS #}
+{% load cms_tags sekizai_tags %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    {# CMS #}
+    {% render_block "css" %}
+
+    <title>{% block title %}Code for Life{% endblock %}</title>
+    <meta name="msapplication-config" content="none"/>
+
+    {% block head %}
+    {% endblock head %}
+
+    {% block css %}
+    {% cache 3600 fragment-base_new-css %}
+    <link href='{% static "portal.css" %}' type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Baloo" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+    {% endcache %}
+    {% endblock css %}
+
+    {% cache 3600 fragment-base_new-javascript %}
+    <script type='text/javascript' src='{% static "portal/js/lib/jquery.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/jquery-ui.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/modernizr.custom.79639.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/common.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/deleteWorkspaces.js" %}'></script>
+    {% endcache %}
+    <script type="text/javascript" src='{% static "portal/js/jquery.placeholder.js" %}'></script>
+
+    {% block check_user_status %}
+    <script>
+        // Set to 'TEACHER', 'SCHOOL_STUDENT', 'INDEPENDENT_STUDENT', or 'UNTRACKED'
+        USER_STATUS = '{{ user|get_user_status }}';
+        DEVELOPER = '{{ user|is_developer }}' === 'True';
+        BETA = '{{ request|has_beta_access }}' === 'True';
+    </script>
+    {% endblock check_user_status %}
+
+    {% block scripts %}
+    {% endblock scripts %}
+
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+
+        ga('create', 'UA-49883146-1', 'auto');
+        ga('send', 'pageview');
+
+    </script>
+
+    <script src="/static/portal/js/riveted.min.js"></script>
+    <script>riveted.init();</script>
```

Comment:
You only need one set of 
`<script>...</script>`
tags, so where you have inline JS, gather it all together inside one script block.  You should probably gather all your 
`<script src="..."></script>`
together as well.

Also, most of it should go at the end of the document, so it's non-blocking.